package.json

```diff
@@ -18,7 +18,7 @@
     "docker-lint": "LINT_COMMAND=lint docker-compose -f docker-compose-lint.yml up --abort-on-container-exit",
     "docker-lint:fix": "LINT_COMMAND=lint:fix docker-compose -f docker-compose-lint.yml up --abort-on-container-exit",
     "docker-test:all": "docker-compose -f docker-compose-test.yml up --abort-on-container-exit",
-    "docker-remove-all": "bash ./scripts/docker-remove-all.sh ch ch-dev-dep-test cd-testv1 code-hammers",
+    "docker-remove-all": "bash ./scripts/docker-remove-all.sh ch ch-dev-dep code-hammers",
```

Comment:
[⛏️ Nitpick]
Edge case - if someone cloned the repo into a folder not named 'code-hammers' the volumes created when running any of the compose files will be prefixed by the folder name and not 'code-hammers' - so this script will not remove those volumes. To be safe we could prefix the volumes in each compose file:
eg. `code-hammers_node_modules` & `code-hammers_client_node_modules`
but for most that will result in ugly volume names like `code-hammers_code-hammers_node_modules`
could pick a shorter identifier to filter volumes by like `ch-vol` or something

Reply:
Good observation but in this case, I'm going to rely on the fact that people clone the repo down as it is, and if they don't then the only side effect is that they lose the convenience of removing volumes with this script. They can pay the penance and manually remove them for their cloning transgressions.